modules/codemod/lib/v8/promoteComponentsToTesting.ts

```diff
@@ -0,0 +1,98 @@
+import {Transform, ImportDeclaration, ASTPath} from 'jscodeshift';
+
+const specifiersToMove = [
+  'ComponentStatesTable',
+  'permutateProps',
+  'ComponentStatesTableProps',
+  'PropCombination',
+  'PropDeclaration',
+  'Props',
+  'PropsDeclaration',
+  'convertToStaticStates',
+  'StaticStates',
+];
+
+// 1. Gather all import specifiers from @workday/canvas-kit-labs-react/common & @workday/canvas-kit-react/common
+// 2. Filter through all imports from @workday/canvas-kit-labs-react/common and gather specifiers we care about
+// 4. If no existing import, create new import with specifiers and insert before closest canvas kit import
```

Comment:
Unless something terrible happened to 3️⃣.
```suggestion
// 3. If no existing import, create new import with specifiers and insert before closest canvas kit import
```